FirefoxPrivateNetworkVPN/ViewControllers/UpdateRequiredViewController.xib

```diff
@@ -81,15 +81,15 @@
                     <nil key="highlightedColor"/>
                 </label>
                 <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" usesAttributedText="YES" lineBreakMode="wordWrap" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="wDU-CA-TDs">
-                    <rect key="frame" x="179.5" y="625" width="665" height="22"/>
+                    <rect key="frame" x="179.5" y="625.5" width="665" height="21.5"/>
                     <constraints>
                         <constraint firstAttribute="width" relation="lessThanOrEqual" constant="665" id="lcf-GJ-ZZh"/>
                     </constraints>
                     <attributedString key="attributedText">
                         <fragment content="Please update the app before you continue to use the VPN">
                             <attributes>
                                 <color key="NSColor" name="custom_grey40"/>
-                                <font key="NSFont" size="15" name="Inter-Regular"/>
+                                <font key="NSFont" metaFont="system" size="15"/>
```

Comment:
Why do we change font here?

Reply:
I've removed unnecessary paragraph style and set up correct fonts.